.gitignore

```diff
@@ -7,3 +7,4 @@ tmp
 *.lst
 *.o
 *.obj
+build
```

Comment:
How did you got the `build` directory?

Reply:
The `build` directory is the default directory for Gradle for (temporary) build artifacts. I use it sometimes for all kind of generated artifacts. In the Epsilon VS code extension it is the default directory for generated compiler artifacts. That way it does not get covered by the git change detection mechanism (e.g., when I'm trying it out at the Gamma project).

But may be, we should give the generation directory another default name, e.g. "gen" or "eag-gen". What do you think?
In the final version, the Epsilon/Gamma/EAG VS code extension is intended to be used at any project having the generator carrying with it (im Bauch haben). So, we need a setting option (which is already there, in fact) where the generated artifact has to be written to in the local project context.